<launch>
  <arg name="pkg" default="morai_script"/>

  <!-- 상태 퍼블리셔: /gps /imu /Competition_topic -> /status -->
  <node pkg="$(arg pkg)" type="status.py" name="status_node" output="screen">
    <!-- 필요하면 입력 토픽 파라미터 추가 -->
    <!-- <param name="gps_topic_in" value="/gps"/> -->
    <!-- <param name="imu_topic_in" value="/imu"/> -->
    <!-- <param name="status_topic_in" value="/Competition_topic"/> -->
  </node>

  <!-- 전역 경로/웨이포인트 퍼블리셔 -->
  <node pkg="$(arg pkg)" type="path.py" name="path_publisher" output="screen">
    <!-- 예: <param name="path_file" value="$(find your_pkg)/paths/main.txt"/> -->
  </node>

  <!-- 경로 MUX: /global_waypoints_src + (선택 입력들) -> /global_waypoints -->
  <node pkg="$(arg pkg)" type="path_mux.py" name="path_mux" output="screen">
    <param name="global_src_topic" value="/global_waypoints_src"/>
    <param name="mux_out_topic"    value="/global_waypoints"/>
    <param name="lattice_kph"      value="25.0"/>
  </node>
</launch>

